lib/lumberjack/client.rb

```diff
@@ -39,7 +41,7 @@ def connect
 
     public
     def write(elements)
-      @socket.write_sync(elements)
+      @socket.write_sync(elements, :json => @opts[:json])
```

Comment:
Just to be consistent, If `write_sync` support specifying a format on each transmission.
Either we:

1. We specify the format at creation time in our `Socket` class, since in reality once a client is made he cannot change the frame format.
2. We add an option hash to the `#write` method that a user can override.

1. is probably easier to reason with.